cobalt/build/get_build_id_test.py

```diff
@@ -21,9 +21,11 @@
 import tempfile
 import unittest
 
+from cobalt.build import build_info
 from cobalt.build import get_build_id
 
-_TEST_BUILD_NUMBER = 1234 + get_build_id.COMMIT_COUNT_BUILD_NUMBER_OFFSET
+_COMMIT_COUNT_BUILD_NUMBER_OFFSET = build_info.COMMIT_COUNT_BUILD_ID_OFFSET
```

Comment:
Why create a local var? Generally I'd prefer keeping the module info with variables, but if you think it hurts readability a lot somehow I'll leave it up to you

Reply:
done